tools/wlr-assets/src/bld_cfg.rs

```diff
@@ -0,0 +1,41 @@
+use path_absolutize::*;
+use std::path::Path;
+
+pub struct LibsConfig {
+    lib_paths: Vec<String>,
+    libs: Vec<String>,
+}
+
+impl LibsConfig {
```

Comment:
This struct is duplicated on `python/tools/wlr-libpython/build.rs`. Can we reuse the methods and structs from this crate?